src/data-card-advanced.tsx

```diff
@@ -0,0 +1,498 @@
+                                </Text>
+                                <Text2
+                                    aria-label="Texto"
+                                    color={vars.colors.textPrimary}
+                                    truncate={subtitleLinesMax}
+                                    as="p"
+                                    regular
+                                    hyphens="auto"
+                                >
+                                    {subtitle}
+                                </Text2>
+                            </Stack>
+                        </Stack>
+                    </header>
+                )}
+
+                {description && (
+                    <Text2
+                        truncate={descriptionLinesMax}
+                        as="p"
+                        regular
+                        color={vars.colors.textSecondary}
+                        hyphens="auto"
+                        aria-label="Texto"
+                    >
+                        {description}
+                    </Text2>
+                )}
+            </Stack>
+        </div>
+    );
+};
+
+type CardFooterProps = {
+    button?: RendersNullableElement<typeof ButtonPrimary>;
+    footerImage?: RendersNullableElement<typeof Image>;
+    footerText?: string;
+    footerTextLinesMax?: number;
+    buttonLink?: RendersNullableElement<typeof ButtonLink>;
+};
+
+const CardFooter: React.FC<CardFooterProps> = ({
+    button,
+    footerImage,
+    footerText,
+    footerTextLinesMax,
+    buttonLink,
+}) => {
+    const {isMobile, isTabletOrBigger} = useScreenSize();
+
+    const hasButton = !!button;
+    const hasFooterImage = !!footerImage;
+    const hasFooterText = !!footerText;
+    const hasButtonLink = !!buttonLink;
+    const hasAllItens = hasButton && (hasFooterImage || hasFooterText) && hasButtonLink;
+
+    const footerCondition = hasAllItens && isMobile;
+
+    const flexDirection = footerCondition ? 'column' : 'row';
+    const marginRight = footerCondition ? '' : 'auto';
+    const alignItems = footerCondition ? 'start' : 'center';
+    const marginTop = hasAllItens ? '8px' : '16px';
+    const marginTopTabletOrBigger = isTabletOrBigger ? '8px' : '';
+    const marginTopButton = isMobile ? '16px' : '8px';
+    const maxWidth = hasButtonLink && !hasAllItens ? '178px' : '';
+
+    return (
+        <div>
+            <div style={{marginLeft: -24, marginRight: -24}}>
+                <Divider />
+            </div>
+
+            <div
+                className={styles.actions}
+                style={{flexDirection, alignItems, marginTop: marginTopTabletOrBigger}}
+            >
+                {hasButton && (
+                    // eslint-disable-next-line jsx-a11y/no-static-element-interactions
+                    <div
+                        aria-label="button"
+                        tabIndex={-1}
+                        onClick={(event) => event.stopPropagation()}
```

Comment:
I guess this is a hack to prevent that clicking on the button would trigger the card onPress isn't it?
Instead of doing this, try to avoid nesting clickable areas

Reply:
turns out the whole card is clickable

Reply:
That's an accesibility issue and the CI a11y checks are warning you. Whe can't have nested clickable areas. You should divide the card in 2 zones: the clickable area (onPress) and the buttons area

Reply:
it turns out that the whole card is clickable, including the area around the buttons